USAGE_android.md

```diff
@@ -393,6 +861,23 @@ adb shell am start -n "com.lunarg.gfxreconstruct.replay/android.app.NativeActivi
 
 If `gfxrecon-replay` was built with Vulkan Validation Layer support, `VK_LAYER_KHRONOS_validation` can be enabled and disabled in the same manner as `VK_LAYER_LUNARG_gfxreconstruct`
 
+
+#### Using the Replay Command
```

Comment:
Could you also document here the "-p" option for playing files that are on the host system ("local")?

Reply:
I have never used that, but I now has a small section mentioning this.